frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -17,6 +17,7 @@ export const listPage = {
       });
   },
   clickCreateYAMLbutton: () => {
+    cy.get(`[data-test="item-create"]`, { timeout: 180 }).should('be.visible');
```

Comment:
Is there a reason not to use the default timeout?

Reply:
@spadgett  Good question, didn't take default timeout into consideration. default timeout should also be ok